### PR TITLE
Add FAQ

### PR DESCRIPTION
**Pull request by rxin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,3 +136,70 @@ python3 -m pip install --user --upgrade twine
...
+
+## FAQ
+
+### What's the project's status?
+This project is currently in beta and is rapidly evolving.
+You should expect the following differences:
+
+ - some functions may be missing (see the [Contributions](#Contributions) section)
+
+ - some behaviour may be different, in particular in the treatment of nulls: Pandas uses
```
Comment (tdliu) [inline README.md]: lower case p for pandas

File: README.md [hunk 2]
```diff
@@ -155,3 +136,70 @@ python3 -m pip install --user --upgrade twine
...
+   Not a Number (NaN) special constants to indicate missing values, while Spark has a
+   special flag on each value to indicate missing values. We would love to hear your use
+   case if you find differences.
```
Comment (tdliu) [inline README.md]: "We would love to hear from you if you come across any discrepancies"?

File: README.md [hunk 3]
```diff
@@ -155,3 +136,70 @@ python3 -m pip install --user --upgrade twine
...
+### Should I use PySpark's DataFrame API or Koalas?
+
+If you are already familiar with pandas and want to leverage Spark for big data, we recommend
+using Koalas. If you are learning Spark from ground up, we recommend you start with PySpark's API.
+
+### How can I request support for a method?
+
+File a GitHub issue: https://github.com/databricks/koalas/issues
+
+Databricks customers are also welcomed to file a support ticket to request a new feature.
```
Comment (tdliu) [inline README.md]: welcomed --> welcome

File: README.md (hunk 1 above)
Comment (tdliu) [inline README.md]: For consistency, "behaviour" --> "behavior"

File: README.md [hunk 4]
```diff
@@ -155,3 +136,70 @@ python3 -m pip install --user --upgrade twine
...
+[missing directory](https://github.com/databricks/koalas/tree/master/databricks/koalas/missing).
+In most cases, it is very easy to add new functions by simply wrapping the existing pandas or
+Spark functions. Pull requests welcome!
+
+### Why a project (vs putting this in Apache Spark itself)?
+
+Two reasons:
+
+1. We want a venue in which we can rapidly iterate and make new releases. The overhead of making
+a release as a separate project is miniscule (in the order of minutes), whereas as part of Spark
```
Comment (tdliu) [inline README.md]: "The overhead of making a release as a separate project is minuscule (in the order of minutes). A release on Spark takes a lot longer (in the order of days)"

"miniscule" --> "minuscule"

**Comment by rxin** [comment]
Merging in. Thanks @tdliu.

File: README.md [hunk 5]
```diff
@@ -155,3 +136,70 @@ python3 -m pip install --user --upgrade twine
...
+
+File a GitHub issue: https://github.com/databricks/koalas/issues
+
+Databricks customers are also welcome to file a support ticket to request a new feature.
+
+### How is Koalas different from Dask?
+
+Different projects have different focuses. Spark is already deployed in virtually every
+organization, and often is the primary interface to the massive amount of data stored in data lakes.
+Koalas draws design inspirations from Dask, and aims to make the transition from pandas to Spark
```
Comment (thunterdb) [inline README.md]: While we are inspired by dask, I would not say that we have design inspiration from it. It is pretty different under the hood.

File: README.md [hunk 6]
```diff
@@ -155,3 +136,70 @@ python3 -m pip install --user --upgrade twine
...
+easy for data scientists.
+
+### How can I contribute to Koalas?
+
+Please create a GitHub issue if your favorite function is not yet supported.
```
Comment (thunterdb) [inline README.md]: Make sure you the name also reflects precisely which function you want to implement, such as `DataFrame.fillna` or `Series.dropna`. If an open issue already exists and you want do add missing parameters, consider contributing to that issue instead.

File: README.md [hunk 7]
```diff
@@ -155,3 +136,70 @@ python3 -m pip install --user --upgrade twine
...
+lot longer (in the order of days)
+
+2. Koalas takes a different approach that might contradict Spark's API design principles, and those
+principles cannot be changed lightly given the large user base of Spark. A new, separate project
+provides an opportunity for us to experiment with new design principles.
+
+### How do I use this on Databricks?
+
+Databricks Runtime for Machine Learning has the right versions of dependencies setup already, so
+you just need to install Koalas from pip when creating a cluster.
```
Comment (thunterdb) [inline README.md]: + link to section

File: README.md [hunk 8]
```diff
@@ -155,3 +136,70 @@ python3 -m pip install --user --upgrade twine
...
+### How can I contribute to Koalas?
+
+Please create a GitHub issue if your favorite function is not yet supported.
+
+We also document all the functions that are not yet supported in the
+[missing directory](https://github.com/databricks/koalas/tree/master/databricks/koalas/missing).
+In most cases, it is very easy to add new functions by simply wrapping the existing pandas or
+Spark functions. Pull requests welcome!
+
+### Why a new project (vs putting this in Apache Spark itself)?
```
Comment (thunterdb) [inline README.md]: instead of

File: README.md (hunk 7 above)
Comment (rxin) [reply README.md]: which section? On Databricks they don't run a command.

**Pull request merged**